content/general/information-technology/information-technology.md

```diff
@@ -0,0 +1,31 @@
+---
+Title: 'Information Technology'
+Description: 'Information Technology is the use of digital technological tools to store and turn data into useful information'
+Subjects:
+  - 'Information Technology'
+  - 'Networking'
+Tags:
+  - 'Information Technology'
+CatalogContent:
+  - 'introduction-to-information-technology'
+  - 'paths/fundamentals-of-cybersecurity'
+---
+
+**Information Technology**, also known as IT, is the use of digital technological tools to store and turn data into useful information. Due to living in a technology-driven world, IT plays a major role in people's lives whether it's fixing a cracked screen on a phone, doing a data transfer from an old computer to another one, or repairing a PC's hardware.
```

Comment:
I think we should broaden the definition a little bit more. Feel free to change the wording. 
```suggestion
**Information Technology**, also known as IT, is the use of digital tools and technologies to store, process, obtain, transmit, and manage useful information. IT encompasses a spectrum of tasks, including designing, implementing, and maintaining systems, networks, databases, and applications. Due to living in a technology-driven world, IT plays a major role in people's lives whether it's fixing a cracked screen on a phone, doing a data transfer from an old computer to another one, or repairing a PC's hardware.
```